### PR TITLE
Add primitive type to ms.parameters

### PR DESCRIPTION
**Pull request by booddu**
Dear DIL maintainers,

Please accept this PR. I understand that it will not be reviewed until I have checked off all the steps below!

### JIRA
- [ ] Add primitive type to ms.parameters

### Description
- [ ] See updated documentation included in this pull request

### Tests
- [ ] Unit tests and AZ test.

### CODE REVIEW

File: cdi-core/src/main/java/com/linkedin/cdi/util/JsonParameter.java [hunk 1]
```diff
@@ -129,6 +129,9 @@ private JsonObject parseParameter(JsonObject paramObject, JsonObject values) {
     JsonObject parsedObject = new JsonObject();
     String name = paramObject.get("name").getAsString();
     switch (type) {
+      case PRIMITIVE:
+        parsedObject.add(name, paramObject.getAsJsonPrimitive("value"));
```
Comment (chris9692) [inline cdi-core/src/main/java/com/linkedin/cdi/util/JsonParameter.java:133]: can you use KEY_WORD_VALUE

File: cdi-core/src/main/java/com/linkedin/cdi/util/ParameterTypes.java [hunk 2]
```diff
@@ -12,6 +12,7 @@
  */
 
 public enum ParameterTypes {
+  PRIMITIVE("primitive"),
```
Comment (chris9692) [inline cdi-core/src/main/java/com/linkedin/cdi/util/ParameterTypes.java:15]: Might be better to use NUMERIC as LIST is actually primitive String.

**Review by chris9692 — CHANGES_REQUESTED** [review]

File: docs/parameters/ms.parameters.md [hunk 3]
```diff
@@ -42,6 +42,8 @@ will be carried as strings.
 
 The following types of parameters are designed: 
 
+- **primitive**: Primitive datatype. for e.g. for parameters like {"id":114027}, {"id":true}, {"id":93.3453}
```
Comment (chris9692) [inline docs/parameters/ms.parameters.md:45]: add  "PRIMITIVE type is for those cases where the value should not be quoted in the final request. While LIST type is for those cases where the value should be quoted."

File: cdi-core/src/main/java/com/linkedin/cdi/util/JsonParameter.java (hunk 1 above)
Comment (booddu) [reply cdi-core/src/main/java/com/linkedin/cdi/util/JsonParameter.java:133]: decided offline to stick with primitive

File: docs/parameters/ms.parameters.md (hunk 3 above)
Comment (booddu) [reply docs/parameters/ms.parameters.md]: done

**Pull request merged**